File: Search for Vowels.py
# Input a string
input_string = input()

# Function to check if a string contains a vowel
def has_vowel(s):
    vowels = "AEIOUaeiou"
    for char in s:
        if char in vowels:
            return True
    return False

# Check if the string contains a vowel
if has_vowel(input_string):
    print("The string contains a vowel.")
else:
    print("The string does not contain any vowel.")

File: Search for Vowels.js
function main (input) {
    /**
     * Write JavaScript code from here
     */
    const vowels = new Set(['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']);

    let foundVowel = false;

    for (let i = 0; i < input.length; i++) {
        if (vowels.has(input[i])) {
            foundVowel = true;
            break;
        }
    }
    
    if (foundVowel) {
        console.log("The string contains a vowel.");
    } else {
        console.log("The string does not contain any vowel.");
    }
}

let input = "";
process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    main(input.trim());
});